tests/test_eventbridge.py

```diff
@@ -246,6 +246,56 @@ def test_filter_resource_with_unknown_target(self):
             eventbridge.ValidEventRuleTargetFilter('event-rule').filter_unsupported_resources(r))
 
 
+def test_event_rule_target_del_managed(test):
```

Comment:
AFAIK there's no way to cause terraform to create AWS-managed event rules, so these don't have any associated terraform. I just used an existing AWS-managed event rule on the account I was testing with.

Let me know if in-file comments explaining the above are desired/appropriate.